tests/js/spec/views/groupDetails/groupEventDetails.spec.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import {initializeOrg} from 'app-test/helpers/initializeOrg';
+import GroupEventDetails from 'app/views/groupDetails/shared/groupEventDetails';
+
+describe('groupEventDetails', () => {
+  let org;
+  let event;
+  let group;
+  let project;
+
+  beforeEach(() => {
+    org = TestStubs.Organization();
+    group = TestStubs.Group();
+    project = TestStubs.Project({organization: org});
+
+    event = TestStubs.Event({
+      size: 1,
+      dateCreated: '2019-03-20T00:00:00.000Z',
+      errors: [],
+      entries: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/issues/${group.id}/`,
+      body: group,
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/issues/${group.id}/events/latest/`,
+      body: event,
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/issues/`,
+      method: 'PUT',
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/events/${event.id}/committers/`,
+      body: {committers: []},
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/events/${event.id}/owners/`,
+      body: {owners: [], rules: []},
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/issues/${group.id}/participants/`,
+      body: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/issues/${group.id}/tags/`,
+      body: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/groups/${group.id}/integrations/`,
+      body: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: '/sentry-apps/',
+      body: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/sentry-app-installations/`,
+      body: [],
+    });
+  });
+
+  it("doesn't load Sentry Apps without being flagged in", () => {
+    const {routerContext} = initializeOrg();
+    const request = MockApiClient.addMockResponse({
+      url: '/sentry-apps/',
+      body: [],
+    });
+
+    mount(
+      <GroupEventDetails
+        group={group}
+        project={project}
+        organization={org}
+        environments={[{id: '1', name: 'dev', displayName: 'Dev'}]}
+        params={{}}
+      />,
+      routerContext
+    );
+
+    expect(request.mock.calls.length).toBe(0);
+  });
+
+  it('loads Sentry Apps when flagged in', () => {
+    const {routerContext} = initializeOrg();
```

Comment:
I *think* this might work? I would not generate an org/project in beforeEach and just use the results from `initializeOrg`

```suggestion
    const {routerContext, organization, project} = initializeOrg({
      organization: {features: ['sentry-apps']},
    });
```